packages/dbml-core/src/export/DbmlExporter.js

```diff
@@ -78,7 +78,7 @@ class DbmlExporter {
         constraints.push(value);
       }
       if (field.note) {
-        constraints.push(`note: '${field.note}'`);
+        constraints.push(`note: '${field.note.replaceAll("'", "\\'")}'`);
```

Comment:
You need to escape the table and enum notes. You can split this functionality to a util method.

Reply:
@nguyenalter done